### ISSUE TITLE
No changes found

### ISSUE THREAD

**Issue by seanturner026** [dag-andersen/argocd-diff-preview#27]
Have configured the github action per the example; however, any PR changes don't appear in the Github PR Comment. Mentioning in case there is a weird bug or edge case going on here. Feel free to close this.

https://github.com/seanturner026/argocd-applicationset/pull/9

My `ApplicationSet` is here: https://github.com/seanturner026/argocd-applicationset/blob/main/clusters/staging/argocd-appset.yaml

Awesome project by the way!

**Comment by dag-andersen** [comment]
Hi, thank you so much for reporting this.

I have now fixed the timeout issue and added support for `kustomize.buildOptions` :)

I have tested it on your branch, and it seems to work now on `v0.0.10` :arrow_down:
```
git clone https://github.com/seanturner026/argocd-applicationset base-branch --depth 1 -q 
git clone https://github.com/seanturner026/argocd-applicationset target-branch --depth 1 -q -b workflow-test
docker run \
   --network host \
   -v /var/run/docker.sock:/var/run/docker.sock \
   -v $(pwd)/output:/output \
   -v $(pwd)/base-branch:/base-branch \
   -v $(pwd)/target-branch:/target-branch \
   -e TARGET_BRANCH=workflow-test \
   -e REPO=seanturner026/argocd-applicationset \
   -e KUSTOMIZE_BUILD_OPTIONS="--load-restrictor LoadRestrictionsNone --enable-helm" \
   dagandersen/argocd-diff-preview:v0.0.10
```

And print the output with
```
cat ./output/diff.md 
```

Let me know if you still experience issues :)

**Comment by seanturner026** [comment]
You legend! Works perfectly.

**Comment by dag-andersen** [comment]
Hi @seanturner026 

I’ve just released version `0.0.18` 🚀  This release removes the `--kustomize-build-options` option. Supporting individual settings like `kustomize.buildOptions` is not sustainable in the long run, which is why I’ve now shifted to allowing users to define any Argo CD configuration through a Helm values.yaml file. The `kustomize.buildOptions` should now be configured in the `values.yaml` file. For more details on how to do this, please refer to the [README](https://github.com/dag-andersen/argocd-diff-preview?tab=readme-ov-file#custom-argo-cd-installation).

Please let me know if you experience any issues or if you have questions about how it works :)

**Comment by seanturner026** [comment]
Cheers thank you!
